Common/TableProducer/centralityTable.cxx

```diff
@@ -183,8 +197,14 @@ struct CentralityTable {
     if (estV0M == 1) {
       float cV0M = 105.0f;
       if (V0MInfo.mCalibrationStored) {
-        float v0m = collision.multV0A() * V0MInfo.mhVtxAmpCorrV0A->GetBinContent(V0MInfo.mhVtxAmpCorrV0A->FindFixBin(collision.posZ())) +
-                    collision.multV0C() * V0MInfo.mhVtxAmpCorrV0C->GetBinContent(V0MInfo.mhVtxAmpCorrV0C->FindFixBin(collision.posZ()));
+        float v0m;
+        if (genName->length() != 0) {
+          v0m = V0MInfo.mMCScale->Eval(collision.multV0M());
```

Comment:
I wonder how efficient this is compared to a C++ formula with some fixed parameters.